elks/arch/i86/lib/homebrew.S

```diff
@@ -0,0 +1,17 @@
+// functions for communicating with homebrew computers (INT 0x88 API)
+
+#define ARG0	2
+
+	.code16
+	.text
+
+	.global	int88
+
+int88:
+	mov	%ds,%dx
```

Comment:
Does `INT 0x88` modify DS? If not, this and line 16 aren't needed. My recommendation would be that INT 0x88 save all registers other than AX (=return value) and flags. ELKS itself (and user land programs) allow BX, CX and DX to be trashed across C function calls, with return value in AX (or DX:AX for longs); the callee must save SI, DI and BP if used.